cmd/bootm.c

```diff
@@ -123,7 +123,7 @@ int do_bootm(cmd_tbl_t *cmdtp, int flag, int argc, char * const argv[])
 			return do_bootm_subcommand(cmdtp, flag, argc, argv);
 	}
 
-#ifdef CONFIG_IMX_HAB
+#if defined(CONFIG_IMX_HAB) && !defined(CONFIG_IMX8M)
```

Comment:
This normally doesn't need to be ifdef'd out.
I wonder if this isn't a side-effect of having `CONFIG_IMX_OPTEE` enabled.  We don't actually support that boot flow (loading OPTEE from u-boot vs. SPL).